drivers/adodb-oci8.inc.php

```diff
@@ -755,13 +755,18 @@ function SelectLimit($sql,$nrows=-1,$offset=-1, $inputarr=false,$secs2cache=0)
 
 			if (is_array($inputarr)) {
 				foreach($inputarr as $k => $v) {
+					if ($this->databaseType == 'oci8po') {
+						$bv_name = ":".array_search($k, array_keys($inputarr));
```

Comment:
To improve performance, since we already loop through $inputarr, I suggest to use a counter variable instead of calling array_search/array_keys in each iteration, i.e.
``` php
$i = 0;
foreach($inputarr as $k => $v) {
    if ($this->databaseType == 'oci8po') {
       $bv_name = ":" . $i++;
```